healthScore/models.py

```diff
@@ -65,6 +71,7 @@ class User(AbstractBaseUser, PermissionsMixin):  # Viewed by User
     is_active = models.BooleanField(default=True)
     is_staff = models.BooleanField(default=False)
     is_patient = models.BooleanField(default=False)
+    is_healthcare_worker = models.BooleanField(default=False)
```

Comment:
Same for this key as well

Reply:
Same idea as the above. Its not getting used now, but once the healthcare worker persona gets added into the workflow it'll get flipped to True in certain cases